client/components/PluginItem.vue

```diff
@@ -6,15 +6,16 @@ const props = defineProps<{
   index?: number
 }>()
 
-const config = $(useServerConfig())
+const config = useServerConfig()
 
-const normalized = $computed((): NuxtPlugin => {
+const normalized = computed((): NuxtPlugin => {
   if (typeof props.plugin === 'string')
     return { src: props.plugin }
   return props.plugin
 })
 
-const shortPath = $computed(() => config ? getShortPath(normalized.src, config.rootDir, true)! : '')
+// TODO: Check if correct
+const shortPath = computed(() => config ? getShortPath(normalized.value.src, config.value?.rootDir ?? '', true)! : '')
```

Comment:
What would be preferred here? The current code or an exists assertion?

```suggestion
const shortPath = computed(() => config ? getShortPath(normalized.value.src, config.value!.rootDir, true)! : '')
```

Reply:
It should check `config.value ?`